packages/integration-karma/test/component/disconnect-callback/index.spec.js

```diff
@@ -14,7 +14,9 @@ afterEach(() => {
     delete window.timingBuffer;
 });
 
-it('should disconnect on the right order (issue #1199 and #1198)', () => {
+// TODO [#1798]: Some incosistency between chrome v/s firefox & Safari
```

Comment:
If this is really a browser quirk, we need a dedicated issue, and we can follow up with implementers. This is pointing to the PR instead.

Reply:
Will do. have a TODO for this in the PR description.